graphql/util.go

```diff
@@ -194,7 +194,11 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 		}
 	}
 
-	tasks.ExecTasks = model.IncludeDependencies(project, tasks.ExecTasks, p.GetRequester())
+	tasks.ExecTasks, err = model.IncludeDependencies(project, tasks.ExecTasks, p.GetRequester())
+	grip.Warning(message.WrapError(err, message.Fields{
+		"message": "error including dependencies for patch",
+		"patch":   patchId,
```

Comment:
preference for "patch_id"

Reply:
I personally don't like typing _ in splunk queries because it's hard to reach. There are a couple of other log lines that do this so I'd like to keep as is